azext_edge/tests/edge/support/create_bundle_int/test_billing_int.py

```diff
@@ -41,10 +41,11 @@ def test_create_bundle_billing(cluster_connection, tracked_files):
     walk_result, bundle_path = run_bundle_command(command=command, tracked_files=tracked_files)
     file_map = get_file_map(walk_result, ops_service)
 
+    # TODO: may not be able to use EdgeApiManager due to the files being in different folders
```

Comment:
note to self: remove this/make a commetn about it in check_custom_resource_files directly